features/karaf/openhab-addons/src/main/feature/feature.xml

```diff
@@ -847,7 +847,10 @@
 
     <feature name="openhab-binding-systeminfo" description="System Info Binding" version="${project.version}">
         <feature>openhab-runtime-base</feature>
-        <bundle start-level="80">mvn:org.openhab.binding/org.openhab.binding.systeminfo/${project.version}</bundle>
+        <bundle dependency="true">wrap:mvn:com.github.oshi/oshi-core/3.13.0</bundle>
```

Comment:
wrapping should not be necessary, this is already an OSGi bundle

Reply:
Played around to solve travis issues. Sadly, still no progress.